src/main/java/org/sagebionetworks/web/client/presenter/ChangeUsernamePresenter.java

```diff
@@ -69,14 +76,15 @@ public void onSuccess(Void result) {
 					
 					@Override
 					public void onFailure(Throwable caught) {
-						if (!DisplayUtils.handleServiceException(caught, globalAppState, authController.isLoggedIn(), view))
-							view.showSetUsernameError(caught);
+						synAlert.handleException(caught);
+						synAlert.showError("Unable to set username. Try again later.");

```

Comment:
remove showError, as this will lose the original error message (displayed in handleException).